torch/distributed/_tensor/api.py

```diff
@@ -402,13 +403,17 @@ def distribute_tensor(
             first rank of each dimension of the `device_mesh`.
 
     Returns:
-        A :class:`DTensor` object
+        A :class:`DTensor(Tensor)` or `XLAShardedTensor(Tensor)` object
     """
```

Comment:
It's worth adding a few more lines in the end of this docstr to explain the behavior for XLA backend, i.e.:
```
.. note:: When initialize the DeviceMesh with the `xla` device_type, `distribute_tensor` would return `XLAShardedTensor` instead. see [link] for more details
```

Reply:
Done